src/themes/DarkTheme.js

```diff
@@ -67,4 +72,35 @@ export default {
   loader: {
     backgroundColor: colors.brand,
   },
+
+  dropdown: {
+    backgroundColor: colors.tuna,
+    shadowColor: shadows.shadowColor,
+    titleColor: colors.white,
+    borderColor: colors.mako,
+    borderColorLight: colors.shark,
+
+    item: {
+      hoverColor: lightenDarkenColor(colors.tuna, -10),
+      activeColor: lightenDarkenColor(colors.tuna, -15),
```

Comment:
Please, keep in mind the convention we're using here.
It should be e.g. `textColor`, `textColorHover`, `textColorActive` instead of `textColor`, `textHoverColor`, textActiveColor`.
It's easier to read.